util/travis.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 
-# make sure pull requests include a changelog
+# Only trigger if we're in a Pull Request
 if [[ x"$TRAVIS_PULL_REQUEST" != x'false' ]]; then
+
+	# Make sure the changelog was updated
     if git diff --quiet master HEAD -- web/changelog.json; then
         echo 'All pull requests must update the changelog.'
         exit 1
     fi
-fi
+
+	# Make sure all changed JSON files are jslint compliant
+	npm install jslint -g
+	git diff --name-only master HEAD | grep .json | while read changedFile; do
```

Comment:
This should only look in `./json/` directory, package.json won't pass lint, and maybe others.